types/mini-css-extract-plugin/index.d.ts

```diff
@@ -6,7 +6,8 @@
 //                 Piotr Błażejewicz <https://github.com/peterblazejewicz>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-import { Configuration, Compiler } from 'webpack';
+
```

Comment:
```suggestion
```

Reply:
I've left the `Compiler` import since it's used in the `apply` function.